Tasks/GitHubReleaseV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -87,5 +87,6 @@
   "loc.messages.FetchCommitDiffError": "An unexpected error occurred while fetching the list of changes.",
   "loc.messages.FetchInitialCommit": "Fetching the initial commit...",
   "loc.messages.FetchInitialCommitSuccess": "Found the initial commit: %s",
-  "loc.messages.FetchInitialCommitError": "An unexpected error occurred while fetching the initial commit."
+  "loc.messages.FetchInitialCommitError": "An unexpected error occurred while fetching the initial commit.",
+  "loc.messages.InvalidActionSet": "Invalid action: %s. Only 'create' or 'edit' or 'delete' actions are allowed. Skipping further steps..."
```

Comment:
only 'create', 'edit' or 'delete' actions are allowed. Need not specify skipping further steps